Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM amazonlinux:2023
+
+RUN dnf update -y && \
+    dnf install -y \
+    make \
```

Comment:
Does `curl` & `wget` come with base image? Probably will need these utils if not